lib/hammer_cli_katello/repository.rb

```diff
@@ -255,6 +262,27 @@ def import_upload_ids(ids)
         }
         resource.call(:import_uploads, params)
       end
+
+      def progress_message(filename)
+        _('Uploading %{filename}') % {:filename => filename}
+      end
+
+      def render_result
+        print_message _('finished')
```

Comment:
Agreed but I'd want to fix it in the other similar code areas as well. I switched this back to work-in-progress.